simplicity-sys/vendor-simplicity.sh

```diff
@@ -1,45 +1,47 @@
 #!/bin/bash
```

Comment:
In 089847766a3ce08527511548c27a67895cb8d691:

If you want this to be nix-compatible we have to replace `/bin/bash` with `/usr/bin/env bash`.